### ISSUE TITLE
PHP5.3 Parse error:  syntax error, unexpected '[', expecting ')' in ./class.phpmaileroauthgoogle.php on line 54

### ISSUE THREAD

**Issue by ildar-aim** [PHPMailer/PHPMailer#664]
I suppose the files should be compatible with PHP5.3

```
aim-server[~/tmp/PHPMailer]$  /opt/phpfarm/inst/php-5.3.29/bin/php -l class.phpmaileroauthgoogle.php
PHP Parse error:  syntax error, unexpected '[', expecting ')' in class.phpmaileroauthgoogle.php on line 54

Parse error: syntax error, unexpected '[', expecting ')' in class.phpmaileroauthgoogle.php on line 54

Errors parsing class.phpmaileroauthgoogle.php
```
```
aim-server[~/tmp/PHPMailer]$  /opt/phpfarm/inst/php-5.3.29/bin/php -l get_oauth_token.php           
PHP Parse error:  syntax error, unexpected T_USE, expecting T_FUNCTION in get_oauth_token.php on line 37

Parse error: syntax error, unexpected T_USE, expecting T_FUNCTION in get_oauth_token.php on line 37

Errors parsing get_oauth_token.php
```
```
aim-server[~/tmp/PHPMailer]$ /opt/phpfarm/inst/php-5.3.29/bin/php -v
PHP 5.3.29 (cli) (built: Oct  8 2014 11:38:06) (DEBUG)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.3.0, Copyright (c) 1998-2014 Zend Technologies
aim-server[~/tmp/PHPMailer]$ 
```

**Comment by SDKiller** [comment]
@ildar-aim 

This particular - not

https://github.com/PHPMailer/PHPMailer/blob/master/class.phpmaileroauthgoogle.php#L4

**Comment by SDKiller** [comment]
@Synchro 

BTW, https://github.com/thephpleague/oauth2-client/blob/master/composer.json#L6

**Comment by SDKiller** [comment]
https://github.com/PHPMailer/PHPMailer/blob/master/README.md#installation--loading

> If you want to use Google's XOAUTH2 authentication mechanism, you need to be running at least PHP 5.4, and load the dependencies listed in composer.json.

**Comment by ildar-aim** [comment]
Thank you for your clarification

**Comment by Synchro** [comment]
Thanks @SDKiller

**Comment by SDKiller** [comment]
Seems they released stable already with 5.5 requirements - change was introduced in https://github.com/thephpleague/oauth2-client/commit/e84bf29023ec96edc27e86795765b0e5bee1fbef

Makes sense to rename 5.4 branch to 5.5 branch ;)

**Comment by Synchro** [comment]
At the slow rate of progress on the xoauth branch (help wanted!), it will probably be up to 5.6 by the time it's released! :)
